server.js

```diff
@@ -13,7 +13,7 @@ const handle = app.getRequestHandler()
 
 const apiPaths = {
     '/api': {
-        // target: 'https://snorkel-backend.herokuapp.com', 
+        //target: 'https://snorkel-backend.herokuapp.com', 
```

Comment:
i would undo this part just since it didn't actually change that way it's not part of the diff